### ISSUE TITLE
Segfault on launch

### ISSUE THREAD

**Issue by thomasdf** [lanoxx/tilda#360]
I get the following output when running tilda

`(tilda:8812): Vte-WARNING **: 21:57:45.964: (../../src/vtegtk.cc:1906):int vte_terminal_match_add_gregex(VteTerminal*, GRegex*, GRegexMatchFlags): runtime check failed: (g_regex_get_compile_flags(gregex) & G_REGEX_MULTILINE)
Segmentation fault (core dumped)`

I'm running Ubuntu 18.04.

**Comment by valessiobrito** [comment]
Debian Unstable + gnome

(gdb) run
Starting program: /usr/bin/tilda 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7fffedc0d700 (LWP 13554)]
[New Thread 0x7fffed40c700 (LWP 13555)]
[New Thread 0x7fffecc02700 (LWP 13556)]
[New Thread 0x7fffdf85c700 (LWP 13558)]
[New Thread 0x7fffdf05b700 (LWP 13559)]
[New Thread 0x7fffde832700 (LWP 13560)]
[New Thread 0x7fffde031700 (LWP 13561)]
[New Thread 0x7fffdd5e6700 (LWP 13562)]

Thread 1 "tilda" received signal SIGSEGV, Segmentation fault.
0x00007ffff5ea4339 in XGetModifierMapping () from /usr/lib/x86_64-linux-gnu/libX11.so.6

**Comment by lifeofguenter** [comment]
similar happening on Debian 10 (buster/testing):

```
~ 🐧 gdb tilda
GNU gdb (Debian 8.2.1-2) 8.2.1
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from tilda...(no debugging symbols found)...done.
(gdb) run
Starting program: /usr/bin/tilda 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[Detaching after fork from child process 3404]
[New Thread 0x7ffff427b700 (LWP 3406)]
[New Thread 0x7ffff3a7a700 (LWP 3407)]
[New Thread 0x7ffff3247700 (LWP 3408)]
[New Thread 0x7ffff2a3f700 (LWP 3409)]
[New Thread 0x7ffff1f58700 (LWP 3411)]
[New Thread 0x7ffff170a700 (LWP 3412)]
[New Thread 0x7ffff0f09700 (LWP 3413)]
[New Thread 0x7fffdbfff700 (LWP 3414)]
[New Thread 0x7fffdb7fe700 (LWP 3415)]
[New Thread 0x7fffdaffd700 (LWP 3416)]
[New Thread 0x7fffda7fc700 (LWP 3417)]
[Detaching after fork from child process 3418]

Thread 1 "tilda" received signal SIGSEGV, Segmentation fault.
0x00007ffff72ab4bb in ?? () from /lib/x86_64-linux-gnu/libX11.so.6
(gdb) 

```

**Comment by lanoxx** [comment]
As far as I can see from the linked Debian issue. This is a Wayland problem. Wayland is not officially supported by tilda.

Version 1.5 to be release soon will contain this fix daa64f579b4d544cb7da614612f1d35c461b43be which allows tilda to run on a Wayland session using XWayland by activating GTKs x11 backend.

This means tilda will be able to run, but you will still have problems activating tilda via Hotkey when a non X11 window has the focus. 

There is a work in progress branch to add D-Bus support to tilda, which might help with this, see branch [wip-dbus](https://github.com/lanoxx/tilda/tree/wip-dbus) in this repository.
